src/main/proto/ga4gh/reads.proto

```diff
@@ -31,37 +31,40 @@ message ReadGroup {
 
   // The ID of the dataset this read group belongs to.
   string dataset_id = 2;
+  
+  // The ID of the read group set this read group belongs to.
+  string read_group_set_id = 3;
 
   // The read group name.
-  string name = 3;
+  string name = 4;
```

Comment:
Sure, I think it's reasonable to 'fix' all of the tag numbers across two major versions of the API.

Leaving space between field numbers is typically not done in my experience. Simplest is just to add fields with increasing tag numbers over time and use a logical organization vertically. Agree, it's slightly bothersome when they go out of order, but it is a small price to pay to avoid data corruption. With this pattern the only thing you really need to remember is "don't reuse a tag number". In my opinion, alternate schemes introduce too much complexity either by process or by subjectivity.

FYI: For field removal you can also use the reserved keyword to enforce that old tags aren't reused https://developers.google.com/protocol-buffers/docs/proto3#reserved